### DISCUSSION TITLE
Delivery of Runtime Package

### DISCUSSION THREAD

**Discussion by nixpt82** [microsoft/AL-Go#1561] [Ideas]
Is a way to run the Get-NavContainerAppRuntimePackage.ps1 from the last release or also create the Runtime Package during the CI/CD or Release?

**Comment by freddydk** [comment]
Currently we haven't build runtime package generation into AL-Go - we are discussing what the right approach is.

**Comment by jonaswre** [comment]
@freddydk Any update on this? 

Is the compiler now capable to create runtimes without a service tier?

Since runtimes don't seem go a way, we need a better way to provide runtimes to our customers...
